plugin.cpp

```diff
@@ -43,6 +43,15 @@ static const char *default_config = QUOTE({
 			"default" : "Random",
 			"order": "1",
 			"displayName": "Asset Name"
+			},
+		"numValues" : {
```

Comment:
This is number of readings to return per plugin_poll call? Please name this accordingly, e.g. "numOutputReadings" or "numReadingsPerPoll" or something like that.